test/integration/test_main.py

```diff
@@ -8,26 +8,16 @@
 import shutil
 import yaml
 import tempfile
-import time
 from contextlib import contextmanager
 import pytest
 
 
 from ansible_runner.exceptions import AnsibleRunnerException
+from test.utils.common import iterate_timeout
```

Comment:
This directory should be reserved for files that are testing `ansible_runner/utils`, not common code shared between tests. Common test code should go in the  appropriate `conftest.py` or maybe a `test/helpers/` or `test/support` directory.

Reply:
Oops, I just realized this is in the `tests/` dir, and not `test/units`. That presents some [another potential issue](https://docs.pytest.org/en/6.2.x/goodpractices.html#tests-outside-application-code) related to `pytest` and imports, but I suppose this is fine for now.